default.nix

```diff
@@ -51,6 +53,9 @@ let
     fonts = nixpkgs.makeFontsConf { fontDirectories = [ nixpkgs.dejavu_fonts ]; };
     docsPackages = if withDocs then [ python3Packages.sphinx ourtexlive ] else [];
 
+    # TODO check if both parameters are given! Otherwise barf.
```

Comment:
Perhaps this should be `assert (nixCrossArch == null) == (crossTargetArch == null); nixCrossTools != null;`?